detectron2/utils/events.py

```diff
@@ -213,7 +221,7 @@ def _get_eta(self, storage) -> Optional[str]:
             eta_string = None
             if self._last_write is not None:
                 estimate_iter_time = (time.perf_counter() - self._last_write[1]) / (
-                    iteration - self._last_write[0]
+                        iteration - self._last_write[0]
```

Comment:
Is this how  they handle indentation in the rest of the repo?

Reply:
will be taken care by linter